tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -1726,3 +1726,291 @@ def test_parse_tribal_section_4_bad_quarter(tribal_section_4_bad_quarter, dfs):
         "representing the Calendar Year and Quarter formatted as YYYYQ"
 
     Tribal_TANF_T7.objects.count() == 0
+
+
+@pytest.fixture
```

Comment:
Is there a way we can parametrize a test related to `EDUCATION_LEVEL, CITIZENSHIP_STATUS, COOPERATION_CHILD_SUPPORT`? Similar to what Mo did in [this](https://github.com/raft-tech/TANF-app/pull/2856/files#diff-4ac7be190ea9ca2ff4edd674fa291b2ec9e2aeee675754c41dd558ab5f6aa8f5:~:text=%40-,pytest,-.mark.parametrize).

Reply:
So would I only be able to co-mingle M2 and M3 into one source file because they are active case data and M5 is closed? I took a quick stab at it initially but was getting generic "Data does not match the expected layout".